modules/serverbidBidAdapter.md

```diff
@@ -0,0 +1,44 @@
+# Overview
+
+Module Name: Serverbid Bid Adapter
+
+Module Type: Bid Adapter
+
+Maintainer: jgrimes@serverbid.com, jswart@serverbid.com
+
+# Description
+
+Connects to Serverbid for receiving bids from configured demand sources.
+
+# Test Parameters
+```javascript
+    var adUnits = [
+        {
+            code: 'test-ad-1',
+            sizes: [[300, 250]],
+            bids: [
+                {
+                    bidder: 'serverbid',
+                    params: {
+                        networkId: '9969',
+                        siteId: '45678'
```

Comment:
Are these params still valid for testing bid responses? Currently getting a `400 Bad Request` from your endpoint with the message `Out of 0 placements on the request, none were valid`. I using these to make sure your `interpretResponse` function is working correctly. That may require updating due to #1748

Reply:
Ah, those aren't valid. But I just updated this PR with ones that are.